### PR TITLE
Frontend scaffolding

### LINKED ISSUE #4: Create Frontend Scaffolding

### PR DESCRIPTION
**Pull request by davidebriani**
- Write versions of used frontend tools in `.tool-versions`
- Bootstrap `frontend` folder with create-react-app
- Setup linting and formatting
- Setup Typescript
- Setup react-bootstrap
- Setup Relay
- Setup i18n and translations with formatjs
- Setup routing and navigation with react-router
- Setup Docker env and CI workflow

Closes #4

### CODE REVIEW

File: .github/workflows/backend-test.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ on:
     - 'backend/**'
     - '.github/workflows/backend-test.yaml'
     branches:
-    - 'master'
+    - 'main'
```
Comment (Pavinati) [inline .github/workflows/backend-test.yaml:10]: Not sure this is the right PR to fix the backend CI

File: frontend/src/index.scss [hunk 2]
```diff
@@ -0,0 +1 @@
+@import "~bootstrap/scss/bootstrap.scss";
```
Comment (Pavinati) [inline frontend/src/index.scss:1]: this file is never imported

File: frontend/src/i18n/index.tsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
+import { createContext, useContext, useMemo, useState } from "react";
+import { IntlProvider } from "react-intl";
+
+import en from "./langs-compiled/en.json";
+import it from "./langs-compiled/it.json";
```
Comment (Pavinati) [inline frontend/src/i18n/index.tsx:5]: this file is empty for now, I think we're better of removing it entirelly than just have an italian option that doesn't change the UI

**Review by Pavinati — CHANGES_REQUESTED** [review]

**Review by Pavinati — APPROVED** [review]

File: docker-compose.yml [hunk 4]
```diff
@@ -23,5 +23,16 @@ services:
     restart: on-failure
     depends_on:
       - postgresql
+
+  edgehog-frontend:
+    image: edgehog-device-manager/edgehog-frontend:snapshot
```
Comment (bettio) [inline docker-compose.yml:28]: `image: edgehogdevicemanager/...`

**Review by bettio — CHANGES_REQUESTED** [review]

**Review by bettio — APPROVED** [review]

**Pull request merged**